packages/conf-openblas/conf-openblas.0.2.0/files/centos_install.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+yes | yum update
```

Comment:
You can do `yum update -y` and `yum install -y git gcc gcc-c++ gcc-gfortran make kernel-devel`

Reply:
ok

Reply:
I think also the yum steps will need sudo